.gitignore

```diff
@@ -8,3 +8,4 @@ yarn.lock
 examples/cordova-sample/MyApp/node_modules
 examples/cordova-sample/MyApp/platforms
 examples/cordova-sample/MyApp/plugins
+package-lock.json
```

Comment:
I keep having to ignore this, it's generated each time, so let's ignore it officially.